sql/src/main/java/io/crate/execution/engine/pipeline/ProjectionToProjectorVisitor.java

```diff
@@ -228,7 +228,8 @@ public Projector visitOrderedTopN(OrderedTopNProjection projection, Context cont
                 numOutputs,
                 OrderingByPosition.arrayOrdering(orderByIndices, projection.reverseFlags(), projection.nullsFirst()),
                 projection.limit(),
-                projection.offset()
+                projection.offset(),
+                10_000
```

Comment:
Maybe we could extract this into a constant which includes some documentation about it? (Why `10_000` was chosen)

Reply:
yep, done. Also moved the CHANGES entry in the fixes section as we'll backport this. Thanks.